### PR TITLE
Feat/genesis

### LINKED ISSUE #1180: Feature: Add support for Genesis Cloud
Genesis cloud focuses on GPU instances and it would be a valuable addition to the portfolio of supported cloud providers by Claudie.

### Description
TF provider [available](https://registry.terraform.io/providers/genesiscloud/genesiscloud/latest/docs) (do not confuse it with Genesys Cloud).

### Exit criteria

- [x] Genesis cloud support added
- [x] E2E tests
- [x] Docs updates

### PR DESCRIPTION
**Pull request by Despire**
Closes #1180.

This change also introduces `PersistentKeepalive = 30` for wiregaurd on dynamic nodepools. As with Genesis cloud the VMs are behind a NAT. Resulting in a more "chatty network".

futher since the disk cannot be referenced in the startup scripts (as with other cloud providers), a workaround had to be used.

https://github.com/berops/claudie/blob/d6f4c89997abe486d2f01fb4b36d9d9d1d426ed4/services/terraformer/templates/genesiscloud/node.tpl#L70-L100

### CODE REVIEW

**Comment by cloudziu** [comment]
Hey @Despire thanks for the PR. I wanted to ask did you run `make crd` after updating the `InputManifest`?

**Comment by Despire** [comment]
> did you run make crd

yes 👍

File: services/terraformer/server/main.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultPrometheusPort = "9090"
+	defaultPrometheusPort = "9097"
```
Comment (cloudziu) [inline services/terraformer/server/main.go:28]: This change is meant to be persistant, or only for testing? 😄

**Review by cloudziu — APPROVED** [review]
LGTM 👍 Only one comment

File: services/terraformer/server/main.go (hunk 1 above)
Comment (Despire) [reply services/terraformer/server/main.go:28]: Only testing 👍

**Pull request merged**